README.md

```diff
@@ -130,11 +147,20 @@ env:
    URL_IN_DESCRIPTION: true
 ```
 
+### Set a publish date for the episode
+
+By setting `SET_PUBLISH_DATE`, the new episode can be scheduled for publishing the episode on the date that the youtube video is uploaded. Please note that the scheduling will work if `SAVE_AS_DRAFT` is not set, because Anchofm doesn't store publish date for draft episodes.
+If `SET_PUBLISH_DATE` is not set, then Anchorfm will choose the current date for publishing.
+```yaml
+env:
+   SET_PUBLISH_DATE: true
+```
 
 # Credits
 
-[@thejoin](https://github.com/thejoin95) & [@wabri](https://github.com/wabri)
+[@thejoin](https://github.com/thejoin95)
 
```

Comment:
I suugest we add the names of those who had a significant contribution

Reply:
Have other people had significant contributions beside those two, you and me?
I suggest we add them in separate PR

Reply:
@matevskial you make a very valid point. This idea is unrelated to this PR
I've already opened an issue regarding this https://github.com/Schrodinger-Hat/youtube-to-anchorfm/issues/45